calcom/cal.com
# Feat Booking Limits

## What does this PR do?

UI

![CleanShot 2022-10-04 at 12 56 07](https://user-images.githubusercontent.com/55134778/193813865-23d9185d-2e05-4d69-a1ea-1e0e3775c613.gif)

Backend Stuff
https://www.loom.com/share/d6c01ec823554d2094b377dda87449e7


Fixes #2684 

## Type of change




## How should this be tested?
* run tests
* add limits 
* check bookings are blocked
